tasks/node/saucelabs.js

```diff
@@ -1,366 +1,224 @@
 'use strict';
-
```

Comment:
The original file was renamed to [remotetest.js](https://github.com/Financial-Times/polyfill-service/pull/1046/files#diff-2b56ca1e2aba3b8dc7f088ac145770d2). 

This file now contains the configuration for running the tests on SauceLabs